### DISCUSSION TITLE
Does my views count for author?

### DISCUSSION THREAD

**Discussion by rrondo** [FreeTubeApp/FreeTube#6235] [Q&A] [answered]
Hi! Simple question: When I watch a video through FreeTube, does my view count toward YouTube's view statistics? Does the channel author benefit in any way from my view?

Thanks!

**Comment by absidue** [comment, marked as answer]
No, YouTube's view tracking is part of their other privacy invasive tracking code, that FreeTube doesn't download or run. FreeTube does not replicate that, firstly as explicitly making it easier to track you would defeat the privacy friendly goal of FreeTube and secondly could potentially open the door to claims about FreeTube artificially inflating view counts.
